### DISCUSSION TITLE
Backend utilizing aws `fargate`

### DISCUSSION THREAD

**Discussion by shunkakinoki** [vercel/vercel#4575] [Ideas] [answered]
It is currently under my knowledge that vercel runs on the AWS lambda as the backend.
Personally, I depend on docker containers to run my software serverlessly, and although I concede the fact that there are many options to overcome the constraints through vercel along with aws lambda, a container serverless option would be certainly really awesome to remove many headaches. I know that the support was dropped some time ago regarding docker builds, but is it on the roadmap eventually to support them in the near future?

**Comment by mcsdevv** [comment, marked as answer]
Hello @shunkakinoki, we have no plans to offer the use of containers ever again, primarily due to the limitations they put on performance. Can you tell us more about your use case?
